lib/pom.xml

```diff
@@ -14,9 +14,8 @@
 
     <properties>
         <version.model>0.1.0-SNAPSHOT</version.model>
-
-        <version.core.lightblue>1.11.0</version.core.lightblue>
-        <version.mongo.lightblue>1.11.0</version.mongo.lightblue>
+        <version.core.lightblue>1.12.0-SNAPSHOT</version.core.lightblue>
+        <version.mongo.lightblue>1.12.0-SNAPSHOT</version.mongo.lightblue>
```

Comment:
Does this change depend on 1.12.0-SNAPSHOT features? What's the release timeline for 1.12.0? I'm worried a bit with the lightblue versions that we may get out of sync when we go to release the hook. I'm not really sure what lightblue version we should be using when we have dependencies on lightblue components so if you have some general guidelines to share that would be appreciated.